devconf2019-lra/README.adoc

```diff
@@ -6,6 +6,9 @@ Base concept of transaction handling of the LRA specification is https://www.cs.
 Saga pattern is designed to provide transactional guarantees for long running transactions
 which is a good fit for the microservice architecture.
 
+NOTE: The quickstart has been updated since it was originally presented.
```

Comment:
I may not have been clear before but I don't think it's very necessary to add this `NOTE` to this particular conference if the only changes are to the paths but it's not wrong either so just wanted to confirm it was not what I thought was necessary but keeping it is good - thanks!